src/Components/AffectedClustersTable/AffectedClustersTable.js

```diff
@@ -0,0 +1,257 @@
+import React, { useEffect, useState } from 'react';
+import { Link } from 'react-router-dom';
+import { useIntl } from 'react-intl';
+import PropTypes from 'prop-types';
+
+import PrimaryToolbar from '@redhat-cloud-services/frontend-components/PrimaryToolbar';
+import { conditionalFilterType } from '@redhat-cloud-services/frontend-components/ConditionalFilter/conditionalFilterConstants';
+import { EmptyTable } from '@redhat-cloud-services/frontend-components/EmptyTable';
+import { TableToolbar } from '@redhat-cloud-services/frontend-components/TableToolbar';
+
+import { global_success_color_100 as globalSuccessColor100 } from '@patternfly/react-tokens/dist/js/global_success_color_100';
+import { global_danger_color_100 as globalDangerColor100 } from '@patternfly/react-tokens/dist/js/global_danger_color_100';
+import CheckCircleIcon from '@patternfly/react-icons/dist/js/icons/check-circle-icon';
+import ExclamationCircleIcon from '@patternfly/react-icons/dist/js/icons/exclamation-circle-icon';
+import { Card, CardBody } from '@patternfly/react-core/dist/js/components/Card';
```

Comment:
Instead of importing everything directly from the 'dist' folder could you use @babel/plugin-transform-runtime in babel.config.js for transforming
import { X } from 'Y'
to
import { X } from 'Y/X' ?
You already using it for @patternfly/react-icons

Reply:
Referenced this suggestion here https://github.com/RedHatInsights/ocp-advisor-frontend/issues/35. Thank you! 👍